setup.py

```diff
@@ -4,9 +4,12 @@
 # See LICENSE for details.
 
 import setuptools
+import sysconfig
 from Cython.Build import cythonize
 
-
+macros = None
+if sysconfig.get_config_var("abi_thread") == "t":
+    macros = [("GRAALVM_PYTHON", "1"), ("Py_GIL_DISABLED", "1")]
```

Comment:
I was thinking that we don't need this.

Instead, we can just use cibuildwheel and will take care of setting up the build environment.